projects/web-app/src/router/index.js

```diff
@@ -1,6 +1,23 @@
 import { createRouter, createWebHistory } from 'vue-router';
+import sharedRoutes from '@shared/views/_shared-routes';
 import useHead from '@/composables/use-head';
-import routes from './routes';
+import Home from '@/views/home.view';
+
+const routes = [
+  {
+    path: '/',
+    name: 'home',
+    component: Home,
+  },
+
+  ...sharedRoutes,
+
+  // Catch all fallback
```

Comment:
Should it be a good idea to add this to the browser extension routing as well?

Reply:
Thought about it, but of course you can only navigate there with the buttons we give the user. Still, I'll add it for safety.

Reply:
👍